cron.py

```diff
@@ -872,6 +895,48 @@ async def check_hibp():
     LOG.d("Done checking HIBP API for aliases in breaches")
 
 
+def notify_hibp():
+    """
+    Send aggregated email reports for HIBP breaches
+    """
+    for user in User.query.all():
+        new_breaches = (
+            Alias.query.filter(Alias.user_id == user.id)
+            .filter(Alias.hibp_breaches_not_notified_user)
```

Comment:
Didn't know that `.filter(Alias.hibp_breaches_not_notified_user)` is enough for sqlalchemy! I usually wrote a subquery for doing this kind of query but this way is much easier :).